docs/version0.9.md

```diff
@@ -22,12 +22,108 @@
+you want the VM to allocate a larger fraction of memory to the Java heap, set the `-XX:+UseContainerSupport` option on the command line.
+
+The following table shows the maximum Java heap size that gets set, depending on the memory available:
+
+
+| Physical memory `<size>`   | Maximum Java heap size (v0.9.0) |
+|----------------------------|---------------------------------|
+| Less than 1 GB             | 50% `<size>`                    |
+| 1 GB - 2 GB                | `<size>` - 512M                 |
+| Greater than 2 GB          | 75% `<size>`                    |
+
+
+## Specifying the maximum Java heap size as a percentage value
+
+OpenJ9 now supports setting the heap size as a percentage of the physical memory. The following OpenJDK options are recognized and can be
+set for the VM:
+
+- ![Start of content that applies to Java 8 and later](cr/java8plus.png) [-XX:MaxRAMPercentage](add link when topic available) ![End of content that applies to Java 8 and later](cr/java_close_lts.png)
+- ![Start of content that applies only to Java 9 and later](cr/java9plus.png) [-XX:InitialRAMPercentage](add link when topic available) ![End of content that applies only to Java 9 or later](cr/java_close.png)
+
+## Shared classes support for nested jar files
+
+Changes are made to the `com.ibm.oti.shared` API to support nested jar files.
+
+## Direct Dump Reader enabled
+
+Direct Dump Reader (DDR) support is now enabled for the OpenJ9 VM on all platforms. The DDR code enables the VM to read system dump data by using the OpenJ9 Diagnostic Tool
+Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool. If you use the [Eclipse Memory Analyzer Tool (MAT)](https://www.eclipse.org/mat/), you can also analyze OpenJ9 or IBM VMs by installing the DTFJ plugin.
+(Install from the Eclipse Help menu; Install New Software > Work with "IBM Diagnostic Tool Framework for Java" > IBM Monitoring and  Diagnostic Tools > Diagnostic Tool Framework for Java)
+
+## Changes to the `java.lang.String` class
+
+![Start of content that applies only to Java 9 and later](cr/java9plus.png) To match the behavior of OpenJDK, `java.lang.String` no longer has a count field, which changes the way that `String.subString()` works compared to Java 8. `String.subString()` now copies the value array. Similarly `StringBuffer`  and `StringBuilder` do not share the value array with any `String` created by `toString()`.
+
+![End of content that applies only to Java 9 or later](cr/java_close.png)
+
+## Changes to the `SharedClassCacheInfo` class
+
+![Start of content that applies only to Java 10 and later](cr/java10plus.png) `SharedClassCacheInfo.getCacheJVMLevel()` used to return the JVMLEVEL constant that maps to a Java version number, for example **JVMLEVEL_JAVA8**. This call now returns only the Java version number, for example **10** for Java 10.
+![End of content that applies only to Java 10 or later](cr/java_close.png)
```

Comment:
I think we're missing a new line for the java_close.png. Without a new line the image appears at the end of the last sentence.

Reply:
I'll take a look at the staged topic to make sure the images work ok and adjust things if they don't.